pinot-distribution/pom.xml

```diff
@@ -93,6 +93,26 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
do we need to shade any of these?

Reply:
they should be provided from plugins jars